src/features/RecipeEdit/components/RecipeForm.tsx

```diff
@@ -355,7 +276,7 @@ const RecipeForm: React.FC<Props> = ({
                             label="Calories"
                             fullWidth
                             placeholder="Calories Per Serving"
-                            value={draft.calories == null ? "" : draft.calories}
+                            value={draft.calories || ""}
```

Comment:
Why is this one `""` instead of `null` for the fallback?

Reply:
Actually, this is an oversight on my part -- they should all be empty strings. React gets cranky on controlled components when there isn't a default value, and `null` doesn't count: https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable. I'll make them all consistent.